pinot-controller/src/main/java/org/apache/pinot/controller/helix/SegmentStatusChecker.java

```diff
@@ -311,6 +312,9 @@ private void updateSegmentMetrics(String tableNameWithType, TableConfig tableCon
           LOGGER.warn("Segment {} of table {} has no online replicas", partitionName, tableNameWithType);
         }
         nOffline++;
+      } else if (nReplicas == 1) {
```

Comment:
should we check if the segment is configured to be single replica in table config? 
If so, i dont think we should emit any metrics (e.g. only emit this when the nReplicas is smaller than the expected one)